public/hide.js

```diff
@@ -3,4 +3,6 @@ if (!document.querySelector('input.filter__box:checked')) {
   document.getElementById('searchresults').className = 'row searchresults';
   document.getElementById('filtercolumn').className = 'filtercolumn';
   document.getElementById('control--filters').className = 'control control--filters';
+  document.getElementById('fb').className = 'control__button';
```

Comment:
The code in `client/routes/search` doesn't get run on the initial render. This file is run before anything else to avoid the box flashing before it's hidden